stl/inc/sstream

```diff
@@ -326,6 +438,36 @@ protected:
         _Mystate = _State;
     }
 
+#if _HAS_CXX20
+    void _Init_string_inplace(_Mystr&& _Str, int _State) {
+        if ((_State & _Constant) && !(_State & _Noread)) { // readonly, copy the string
+            _Init(_Str.c_str(), _Str.size(), _State);

```

Comment:
I still don't think checking anything related to _State here is correct. The input was an rvalue, therefore we may take its buffer.

Reply:
OK, I'll take its buffer.🤣 But anyway it should be copied.

Reply:
I'll try.